lib/cloud_controller/serializer.rb

```diff
@@ -7,7 +7,13 @@ def serializes_via_json(accessor_method_name)
         string = send("#{accessor_method_name}_without_serialization")
         return if string.blank?
 
-        MultiJson.load string
+        begin
+          MultiJson.load string
+        rescue MultiJson::ParseError
+          logger = Steno.logger('cc.serializer')
+          logger.error("Failed to deserialize #{guid}")
+          { 'corrupted-env': string }
```

Comment:
As these methods are also defined for other fields, returning `corrupted-env` seems wrong to me.

Reply:
Good point.

Reply:
Fixed this, easy fix. As for the others, I'm going to chat with some of my teammates either late this week or early next week about this.